datakind/housd-bright-event-siting-tool
# Hotfix: Remove geometry column from dashboard table

This PR adds an `as.data.frame()` call to convert the raw `sf` table to a regular `data.frame`, stripping out the `geometry` column and avoiding it being displayed in the flexdashboard.

### Suggested review steps

1. Download `frontend/ws3dash.html`
2. Open in browser
3. Verify that the `geometry` column no longer appears
